Ginger/Ginger/ApplicationModelsLib/POMModels/POMWizardLib/LearnWizard/POMObjectsMappingWizardPage.xaml.cs

```diff
@@ -123,8 +122,7 @@ private void Timer_Tick(object sender, EventArgs e)
         private void StopTimer()
         {
             timer.Stop();
```

Comment:
Add null checks and add try catch to handle unexpected error

Reply:
done